### ISSUE TITLE
✨ Display Category-wise Recent Posts

### ISSUE THREAD

**Issue by thecoducer** [nunocoracao/blowfish#669]
I have several categories like Technical, Travel, and Problem Solving for which I want to show at least 5 recent posts from each category on the homepage. Currently, the homepage shows recent posts without any category-wise grouping.

Here is my site: https://thecoducer.com
Repo: https://github.com/thecoducer/blog

The homepage (https://thecoducer.com/) should show the posts category-wise whereas this page (https://thecoducer.pages.dev/articles/) should continue to display all the posts.

How can this be achieved? If anyone can pick this up, it would be of great help. Thanks!

Structure of my content folder:
```
.
├── about
├── categories
│   ├── events
│   ├── opinion
│   ├── technical
│   └── travel
├── jupyter-notebooks
├── posts
│   ├── 2017
│   │   ├── 04
│   │   ├── 07
│   │   ├── 08
│   │   └── 09
│   ├── 2018
│   │   ├── 01
│   │   ├── 02
│   │   └── 10
│   ├── 2019
│   │   ├── 01
│   │   ├── 04
│   │   ├── 06
│   │   ├── 07
│   │   └── 09
│   ├── 2020
│   │   ├── 08
│   │   └── 12
│   ├── 2021
│   │   ├── 01
│   │   └── 04
│   ├── 2022
│   │   ├── 01
│   │   ├── 02
│   │   ├── 03
│   │   └── 12
│   └── 2023
│       └── 03
├── problem-solving
│   ├── 2017
│   │   ├── 08
│   │   └── 09
│   ├── 2018
│   │   ├── 09
│   │   └── 10
│   ├── 2019
```

**Comment by nunocoracao** [comment]
@thecoducer This is not supported atm. You could build an override to the main page and build it yourself. 
However, this seems to be connected to another request https://github.com/nunocoracao/blowfish/issues/619

I will need sometime to implement this but sounds like a nice feature to add

**Comment by thecoducer** [comment]
@nunocoracao I have no idea how to start with it. Can you guide me? So, that I can probably get started with it.

**Comment by nunocoracao** [comment]
Hey @thecoducer I would advice starting in the Hugo docs and get it from there. 
TL;DR - in the root of your project you can create a `layouts` folder, everything you place inside will override the files in the themes you are loading (in the same relative directory). Check the repo for Blowfish webpage and inside the `exampleSite` there are some overrides you can look at as examples. 

Not implementing this on the theme, going with the better reader experience in https://github.com/nunocoracao/blowfish/issues/495

**Comment by thecoducer** [comment]
Thank you @nunocoracao 
But this should be in the theme itself IMHO. 

Just like this:

<img width="898" alt="image" src="https://user-images.githubusercontent.com/24296902/235322252-e07410a9-00bc-44ef-b271-9fb867cac8ef.png">

This will allow visitors to get a quick glimpse of the recent posts in different categories on the homepage. It would be very convenient.

**Comment by nunocoracao** [comment]
Not disagreeing. Right now the theme already supports showing the recent articles across multiple areas. Just doesn't group them. Given that the ticket didn't receive any traction since it was open I am closing it. If you want to open a PR happy to re-open the ticket.
